pkg/resolvconf/resolvconf.go

```diff
@@ -0,0 +1,323 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
```

Comment:
Probably better to show Docker/Moby copyrights too.

Like this: 
https://github.com/containerd/nerdctl/blob/f6f48cd29548226c2102f65c26bb7c807f0eca8d/pkg/inspecttypes/dockercompat/dockercompat.go#L17-L21